projects/wave-core-new/src/public-api.ts

```diff
@@ -72,6 +72,7 @@ export * from './lib/operators/dialogs/raster-vector-join/raster-vector-join.com
 export * from './lib/operators/dialogs/scatterplot-operator/scatterplot-operator.component';
 export * from './lib/operators/dialogs/statistics-plot/statistics-plot.component';
 export * from './lib/operators/dialogs/temporal-raster-aggregation/temporal-raster-aggregation.component';
+export * from './lib/operators/dialogs/raster-type-conversion/raster-type-conversion.component';
```

Comment:
This list was sorted lexicographically so please sort it again.

Reply:
why is there no auto sort if that is important?

Reply:
[426cf2e](https://github.com/geo-engine/geoengine-ui/pull/424/commits/426cf2ee8f30ee0443486e2581c97fe1b701909c)

Reply:
I'm not sure how to do that.